daemon/logic/session.go

```diff
@@ -52,6 +52,17 @@ func (s *Session) Login(ctx context.Context, creds apitypes.LoginCredential) err
 	return s.engine.session.Set(self.Type, self.Identity, self.Auth, creds.Passphrase(), authToken)
 }
 
+// Refresh retrieves the user's identity and updates the stored copy of the session
```

Comment:
Why not just have the daemon handle the username/email/password changes instead of proxying?

The "daemon cache" and it's responsibility should be transparent to the CLI.